qa/rpc-tests/test_framework/mininode.py

```diff
@@ -39,6 +39,7 @@
     zcash_person,
 )
 
+OVERWINTER_PROTO_VERSION = 170003
```

Comment:
This isn't used anywhere. Is there a missing test, or is this preparatory?

Reply:
Preparatory

Reply:
And missing test.